storyblok/storyblok-design-system
# feat(work-1450): adding undo-pictogram and warning-pictogram icons to design system's internal icon list

Adding two new icons to the design system's internal icon list

<img width="653" alt="Screenshot 2023-11-29 at 07 57 28" src="https://github.com/storyblok/storyblok-design-system/assets/7618411/cd998e18-95d7-4d33-b171-d8b0fa5c4021">

<img width="738" alt="Screenshot 2023-11-29 at 09 01 38" src="https://github.com/storyblok/storyblok-design-system/assets/7618411/d24e6ef7-6e7b-4589-a8e0-a37c02d74be0">




## Pull request type

Jira Link: [WORK-1450](https://storyblok.atlassian.net/browse/WORK-1450)



- [x] Other (please describe): Another icon to the list

## How to test this PR

Go to the [icons list](https://storyblok-design-system-git-feat-work-1450-6aade6-storyblok-com.vercel.app/?path=/story/basic-sbicon--storyblok-icons) and check if the new icon is present

<img width="181" alt="Screenshot 2023-11-29 at 08 01 02" src="https://github.com/storyblok/storyblok-design-system/assets/7618411/6bf1661b-9e68-4709-b743-284b046ac356">

<img width="197" alt="Screenshot 2023-11-29 at 09 02 35" src="https://github.com/storyblok/storyblok-design-system/assets/7618411/ee9ff2d1-2509-4e63-806e-248ce5c5f907">

## What is the new behavior?

## Other information


[WORK-1450]: https://storyblok.atlassian.net/browse/WORK-1450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
